code/modules/mob/living/emote.dm

```diff
@@ -263,7 +263,7 @@
 /datum/emote/living/laugh/get_sound(mob/living/user)
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
-		if(H.dna.species.id == SPECIES_HUMAN && (!H.mind || !H.mind.miming))
+		if(H.dna.species.id == SPECIES_HUMAN && !HAS_TRAIT(H, TRAIT_MIMING))
 			if(user.gender == FEMALE)
 				return 'sound/voice/human/womanlaugh.ogg'
 			else
```

Comment:
```
	if(!ishuman(user))
		return
	var/mob/living/carbon/human/joker = user
	if(joker.dna.species.id != SPECIES_HUMAN || HAS_TRAIT(joker, TRAIT_MIMING))
		return
	if(user.gender == FEMALE)
		return 'sound/voice/human/womanlaugh.ogg'
		
	return pick('sound/voice/human/manlaugh1.ogg', 'sound/voice/human/manlaugh2.ogg')
```

Reply:
ok this is a nitpick but sure battlebuddy

Reply:
agreed, so I wrote it for ya pardner